sdks/python/setup.py

```diff
@@ -205,6 +204,51 @@ def generate_protos_first():
                        err.stderr)
 
 
+def generate_external_transform_wrappers():
+  try:
+    sdks_dir = os.path.realpath(
+        os.path.join(os.path.realpath(__file__), '..', '..'))
+    script_exists = os.path.exists(
+        os.path.join(sdks_dir, 'python', 'gen_xlang_wrappers.py'))
+    config_exists = os.path.exists(
+        os.path.join(sdks_dir, 'standard_external_transforms.yaml'))
+    # we need both the script and the standard transforms config file.
+    # at build time, we don't have access to apache_beam to discover and
+    # retrieve external transforms, so the config file has to already exist
+    if not script_exists or not config_exists:
+      generated_transforms_dir = os.path.join(
+        sdks_dir, 'python', 'apache_beam',
+        'transforms', '_external_transforms')
+
+      # if exists, this directory will have at least its __init__.py file
+      if (not os.path.exists(generated_transforms_dir) or
+              len(os.listdir(generated_transforms_dir)) <= 1):
+        message = 'External transform wrappers have not been generated '
+        if not script_exists:
+          message += 'and the generation script `gen_xlang_wrappers.py`'
+        if not config_exists:
+          message += 'and the standard external transforms config'
+        message += ' could not be found'
+        warnings.warn(message)
+      else:
+        warnings.warn(
```

Comment:
Will users see this every time?

Reply:
I don't see it when running `setup.py`, but will reduce log level because this should be `INFO` level at most